src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,24 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int bulldozerYear;
```

Comment:
it's the same property for every Machine, can be moved to class Machine. let's introduce unique properties in each machine